packages/storage/src/mappings/tables.rs

```diff
@@ -399,17 +399,19 @@ impl TryFrom<L1Tx> for ports::types::L1Tx {
 pub enum L1TxState {
     Pending,
     Finalized,
-    Failed,
+    SqueezedOut,
     IncludedInBlock,
+    Failed,
 }
 
 impl From<L1TxState> for i16 {
     fn from(value: L1TxState) -> Self {
         match value {
             L1TxState::Pending => 0,
             L1TxState::Finalized => 1,
-            L1TxState::Failed => 2,
+            L1TxState::SqueezedOut => 2,
```

Comment:
This only applies to blob transactions which cannot fail in execution so every tx that is currently marked as failed is actually squeezed out. I thought this would be the least painful way of adapting the db but I see it could cause issues with the non finalized fragments query so I'll leave the original ordering.